app/javascript/styles/mastodon/components.scss

```diff
@@ -405,7 +405,7 @@
     }
 
     input[type="checkbox"] {
-      display: none;
+      opacity: 0;
```

Comment:
Issue with display:none was that it was causing the checkbox to be skipped when we are cycling the components via tab. Opacity:0 fixes that.